src/app/books.ts

```diff
@@ -2433,12 +2421,208 @@ export const BOOKS : Book[] = [
+				"title": "Fimfiction.net"
+			},
+			{
+				"url": "https://forms.gle/JRbnp3q5Nnp3kmfXA",
+				"title": "Signup Form"
+			}
+		],
+		"tags": [
+			"fimfic", "hardcover", "signup"
+		]
+	}
+	,{
+		"id": 88,
+		"title": "Stardust",
+		"edition": "2",
+		"img": "https://cdn-img.fimfiction.net/story/ddv4-1432482453-100455-medium",
+		"rating": "T",
+		"dateAdded": "2019/04/24",
+		"expiry": null,
+		"authors":[
+			{
+				"name": "Arad",
+				"url": "https://www.fimfiction.net/user/45831/Arad"
+			}
+		],
+		"links": [
+			{
+				"url": "https://www.fimfiction.net/story/100455/stardust",
+				"title": "Fimfiction.net"
+			},
+			{
+				"url": "https://forms.gle/JRbnp3q5Nnp3kmfXA",
+				"title": "Signup Form"
+			}
+		],
+		"tags": [
+			"fimfic", "hardcover", "signup"
+		]
+	}
+	,{
+		"id": 89,
+		"title": "Tales of the Sun",
+		"edition": "1",
+		"img": "https://static.wixstatic.com/media/45d896_4da7b378c7b049479da6e5f7cc74bacf~mv2_d_4335_2941_s_4_2.jpg/v1/fill/w_1125,h_717,al_c,q_85,usm_0.66_1.00_0.01/45d896_4da7b378c7b049479da6e5f7cc74bacf~mv2_d_4335_2941_s_4_2.webp",
+		"rating": "T",
+		"dateAdded": "2019/04/24",
+		"expiry": null,
+		"authors":[
+			{
+				"name": "Ministry of Image",
+				"url": "https://www.ministryofimage.net"
+			}
+		],
+		"links": [
+			{
+				"url": "https://www.fimfiction.net/bookshelf/1161653/tales-of-the-sun",
+				"title": "Fimfiction.net"
+			},
+			{
+				"url": "https://www.ministryofimage.net/product-page/tales-of-the-sun",
+				"title": "Hardcover"
+			}
+		],
+		"tags": [
+			"fimfic", "hardcover", "anthology", "Administrative Angel", "Analemma", "Celestia XVII", "Dune Goddess", "In Celestia We Trust, All Others Pay Cash", "Princess Celestia Hates Tea", "Sisters", "Sunny Skies All Day Long", "Whom the Princesses Would Destroy..."
+		]
+	}
+	,{
+		"id": 90,
+		"title": "Tales of the Moon",
+		"edition": "1",
+		"img": "https://static.wixstatic.com/media/45d896_93527fce20e04eb9b7541454131bb344~mv2_d_4311_2941_s_4_2.jpg/v1/fill/w_1125,h_717,al_c,q_85,usm_0.66_1.00_0.01/45d896_93527fce20e04eb9b7541454131bb344~mv2_d_4311_2941_s_4_2.webp",
+		"rating": "T",
+		"dateAdded": "2019/04/24",
+		"expiry": null,
+		"authors":[
+			{
+				"name": "Ministry of Image",
+				"url": "https://www.ministryofimage.net"
+			}
+		],
+		"links": [
+			{
+				"url": "https://www.fimfiction.net/bookshelf/1161654/tales-of-the-moon",
+				"title": "Fimfiction.net"
+			},
+			{
+				"url": "https://www.ministryofimage.net/product-page/tales-of-the-sun",
+				"title": "Hardcover"
+			}
+		],
+		"tags": [
+			"fimfic", "hardcover", "anthology", "Her Soldiers, We", "Luna, Princess of Space", "Lunangrad", "Music for the Mare in the Moon", "Que Sera, Sera", "Quiet Boy and Moon Horse", "Tenebrae", "The Nightmare Omen", "Thou Goddess"
+		]
+	}
 	/* THIS IS  A SAMPLE TO COPY FROM; DO NOT DELETE
 	,{
 		"id": ###,
 		"title": "",
 		"edition": "",
-		"img": "",
+		"img": "(if using fimfiction, use a -medium link)",
```

Comment:
I'm not sure if I fundamentally agree with using -medium across the board.

Reply:
While I do not think shrinking everything down as much as possible is a top priority, I do think that preventing unnecessary bloat is worthwhile, and -medium fimfic covers are definitely high-res enough for our use case here.

Reply:
As the images are used both in the index listing and the detail page, I think that's a fair call. I think we *could* use a higher res image on the detail page though.